CONTRIBUTING.rst

```diff
@@ -0,0 +1,214 @@
+Contributing
+============
+
+Thanks for contributing to OpenSCM. We are always trying to improve this
+tool, add new users and can do so even faster with your help!
+
+Following the guidelines will help us work together as efficiently as
+possible. When we all work with a common understanding, we can make sure
+that issues are addressed quickly, suggested changes can be easily
+assessed and pull requests can be finalised painlessly.
+
+All contributions are welcome, some possible suggestions include:
+
+-  bug reports (make a new issue and use the template please :D)
+-  feature requests (make a new issue and use the template please :D)
+-  pull requests (make a pull request and use the template please :D)
+-  tutorials (or support questions which, once solved, result in a new
+   tutorial :D)
+-  improving the documentation
+-  blog posts
+
+Please don’t use the repository to have discussions about the results.
+Such discussions are scientific and generally belong in the scientific
+literature, not in a development repository.
+
+
+Ground Rules
+************
+
+As a contributor, it is vital that we all follow a few conventions:
+
+-  Be welcoming to newcomers and encourage diverse new contributors from
+   all backgrounds. See the :ref:`code_of_conduct`.
+-  Create issues for changes and enhancements, this ensures that
+   everyone in the community has a chance to comment
+-  Use our ``.editorconfig`` file (see `Editor
```

Comment:
I don't and i think this can be discouraging, especially since we use `black` so these defaults matter too much anyhow.

Reply:
I removed that as a "requirement"